packages/wrangler/src/api/startDevWorker/ConfigController.ts

```diff
@@ -388,6 +423,7 @@ export class ConfigController extends Controller<ConfigControllerEventMap> {
 	async teardown() {
 		logger.debug("ConfigController teardown beginning...");
 		await this.#configWatcher?.close();
+		await this.#assetsWatcher?.close();
```

Comment:
If the first call rejects, the 2nd won't be called.

We should wrap these in Promise.allSettled

Reply:
ah yes, you're totes right